tools/ctl/ide/ui.go

```diff
@@ -368,7 +369,15 @@ func (e *explorer) makeVerdictGroupNode(vg *rundex.VerdictGroup, percent float32
 }
 
 func (e *explorer) makeRunNode(runid string) *tview.TreeNode {
-	node := tview.NewTreeNode(runid).SetColor(tcell.ColorGreen).SetSelectable(true)
+	var title string
+	if run, ok := e.runs[runid]; ok && run.Type == benchmark.AttestMode {
+		title = fmt.Sprintf("(a)%s", runid)
```

Comment:
ugh I don't love the single-letters, especially for new users, but I guess it's fine for now...

would mainly want to keep them the same length and I can't think of two descriptors that are consistent with our current terminology and maintain that property.

Reply:
Discussed offline, we decided to move the descriptors to the end of the runid and use longer, more descriptive terms.